content/en/network_monitoring/devices/supported_devices.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Supported Devices
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/datacenter-monitoring-dashboards/'
+      tag: 'Blog'
+      text: 'Monitor datacenters and network devices with Datadog'
+    - link: 'https://www.datadoghq.com/blog/network-device-monitoring/'
+      tag: 'Blog'
+      text: 'Introducing Network Device Monitoring'
+    - link: 'https://www.datadoghq.com/blog/diagnose-network-performance-with-snmp-trap-monitoring/'
+      tag: 'Blog'
+      text: 'Monitor and diagnose network performance issues with SNMP Traps'
+---
+
+## Supported devices
+
+The [vendor profiles](#vendor-profiles) section below lists the network monitoring devices that Datadog supports with out-of-the-box profiles. Datadog can also monitor devices not included in this list using a [generic profile](#generic-profile), which collects basic metrics if applicable for those devices.
+
+### Generic profile
+
+The [generic profile][114] collects metrics for all devices not supported by a vendor profile. Metrics include TCP, UDP, IP, and interface metrics such as bandwidth utilization, and volume sent/received.
+
+### Vendor profiles
+
+The following vendor devices are supported with dedicated profiles. If a vendor or device type is supported, but the specific model isn't, you can:
+  - Customize your own YAML profile (see the [NDM troubleshooting FAQ page][1] for guidance).
+  - Create new [profiles][2] to start monitoring new device models.
+
+| Vendor | Config files |
+| ---  | ----------- |
+| 3com | [3com.yaml][3] <br> |
+| TP-Link | [tp-link.yaml][4] <br> |
+| A10 | [a10.yaml][5] <br> |
+| Alcatel-lucent | [alcatel-lucent.yaml][6] <br> |
+| Anue | [anue.yaml][7] <br> |
+| Apc | [_apc.yaml][8] <br>[apc_ups.yaml][9] <br> |
+| [Arista][10] | [arista.yaml][11] <br>[_arista.yaml][12] <br> |
+| [Aruba][13] | [aruba-switch.yaml][14] <br>[_aruba-base.yaml][15] <br>[aruba-access-point.yaml][16] <br>[aruba.yaml][17] <br> |
+| Audiocodes | [audiocodes-mediant-sbc.yaml][18] <br> |
+| Avaya | [avaya.yaml][19] <br> |
+| Avocent | [avocent-acs.yaml][20] <br> |
+| Avtech | [avtech.yaml][21] <br> |
+| [Barracuda][22] | [barracuda.yaml][23]<br> |
```

Comment:
The link to barracuda.yaml didn't render correctly in the preview.

Reply:
ok thanks! I'll take a look.